bin/faucet/src/static/index.js

```diff
@@ -56,6 +78,21 @@ document.addEventListener('DOMContentLoaded', function () {
         }
     }
 
+    function toggleVisibility(clickedPrivate) {
```

Comment:
If this function changes the state of the global variable used in the request, I'd maybe name it something like "onNoteTypeButtonClick".

Reply:
how about something like `toggleNoteTypeButton` and rename the parameter to `createPrivate` or similar? I think it describes better wheat the function does

Reply:
Yes, that name wasn't very descriptive but in the new version we don't need that function anymore.